websockets.md

```diff
@@ -4,9 +4,17 @@ For use cases, when polling would be too resource consuming or not enough "real-
 
 ## Endpoint
 
-`[PlatformAddress]/ws/connector?ClientToken=[ClientToken]&AccessToken=[AccessToken]`
+`[PlatformAddress]/ws/connector`
 
-Note that protocol of the `[PlatformAddress]` should be changed to `ws(s)://`. If `[PlatformAddress]` starts with `http://` then `ws://` should be used. If it starts with `https://` then `wss://` should be used. `ClientToken` and `AccessToken` parameter values should be the same as in case of standard API operations. For further details, consult the [Authentication](guidelines.md#authentication) section.
+Note that protocol of the `[PlatformAddress]` should be changed to `ws(s)://`. If `[PlatformAddress]` starts with `http://` then `ws://` should be used. If it starts with `https://` then `wss://` should be used. 
+
+## Authentication
+
+`Cookie: ClientToken=[ClientToken]; AccessToken=[AccessToken]`
```

Comment:
This is a bit unclear, are those multiple cookies or single cookie? We should define more clearly what's the cookie name and what is the value.

Reply:
Haven't found much other examples how it's handled in other documentations, taken the notation from https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies 
![image](https://user-images.githubusercontent.com/1150547/102888028-4035a380-4458-11eb-8faf-358b9d2b1f5b.png)

Reply:
Consider using a table with columns for cookie name and cookie value. The values should be described same way as they're described in standard POST requests. This string representation of cookies can be turned into example with real demo client token and access token inside instead of placeholders.